server/activate_hooks.go

```diff
@@ -56,6 +59,17 @@ func (p *Plugin) OnActivate() error {
 		}
 	}
 
+	job, cronErr := cluster.Schedule(
+		"BackgroundJob",
+		cluster.JobConfig{Interval: 1 * time.Minute},
+		p.BackgroundJob,
+		p.API,
+	)
+	if cronErr != nil {
+		p.API.LogError("Failed to schedule background job", "err", cronErr)
```

Comment:
Should the code return here?

Reply:
Ah, yeah, for the demo plugin, that makes sense. Good call!